### ISSUE TITLE
Always On Top Button 

### ISSUE THREAD

**Issue by millscale** [redox-os/orbtk#117]
# Always on Top, great for utility apps
A user wants to have one window/widget/application always be on top of other windows.

# Current Window behaviour switches between windows (as expected)
If a Always On Top Button is added to a window, it might be great for apps like the Calculator, that the user will always want to keep on top when doing repetitive tasks. 

# Discussion Points
There will probably have to be a caveat where only one window can be marked as always on top?

Otherwise more than one window can be marked as always on top, and they then join a second layout group where they all behave as expected. (Focus switching between windows)

**Comment by FloVanGH** [comment]
This topic will need investigation. How we could implement such a feature with minifb or winit. And it would be necessary to check this behavior on Redox, Windows, macOS and different Linux desktops.

**Comment by millscale** [comment]
I understand it is a tricky one. I just realised that it was really important, after I read that it was the _second highest feature request_ on Microsoft Feedback Hub. They only added the functionality recently to the Calculator specifically.
https://github.com/microsoft/calculator/issues/78

[Apparently Ubuntu has a pin on top feature](https://github.com/microsoft/calculator/issues/78#issuecomment-470613606) (I don't know Ubuntu well) 

There are security concerns with this as well, as malicious actors can possibly create empty buttons, fake input fields etc. as well, as read in the thread above.

**Comment by millscale** [comment]
I see this in winit/src/window.rs

Ln 153.
`
    /// Whether the window should always be on top of other windows.
    ///
    /// The default is `false`.
    pub always_on_top: bool,`

**Comment by FloVanGH** [comment]
That was fast. Thank you for the research.

**Comment by phillvancejr** [comment]
Just wanted to second this request! If Orbtk gets this functionality I will be using it instead of Flutter or Csharp Avalonia

**Comment by phillvancejr** [comment]
Hello, my fork of emoon's original minifb added always on top (topmost) functionality for MacOS and was merged to master.  With this change and my own clone of this project, on MacOS Orbtk windows can now be always on top using the Window::topmost method when creating the window.  After making changes to emoon's minifb I realized that Orbtk uses your own fork of minifb.  Should I fork your version of minifb and add the topmost functionality there before forking orbtk itself?
